OMPython/OMCSession.py

```diff
@@ -801,20 +801,23 @@ def run_model_executable(cmd_run_data: OMCSessionRunData) -> int:
                 env=my_env,
                 cwd=cmd_run_data.cmd_cwd_local,
                 timeout=cmd_run_data.cmd_timeout,
-                check=True,
+                check=False,
```

Comment:
Getting an exception on non-zero exit code was good. Exceptions for handling control flow in Python are totally fine, and obvious to the user, and hard to miss, I would keep that. (I did not realize that that's what's happening because I did not dig deep enough in the call hierarchy)

The additional logging is good, so I'd keep that too. 👍